### PR TITLE
Move no-signing to RTV exp

### PR DESCRIPTION
**Pull request by calebcordry**
Move branch level No Signing experiment to ExperimentA slot to stop polluting canary logs.

link without whitespace in diff :) https://github.com/ampproject/amphtml/pull/30201/files?w=1

### CODE REVIEW

**Review by estherkim — APPROVED** [review]
Experiments config & e2e changes LGTM

File: extensions/amp-a4a/0.1/amp-a4a.js [hunk 1]
```diff
@@ -870,10 +860,8 @@ export class AmpA4A extends AMP.BaseElement {
    * @return {boolean}
    */
   isInNoSigningExp() {
-    return (
-      getExperimentBranch(this.win, NO_SIGNING_EXP.id) ===
-      NO_SIGNING_EXP.experiment
-    );
+    // eslint-disable-next-line no-undef
+    return NO_SIGNING_RTV;
```
Comment (powerivq) [inline extensions/amp-a4a/0.1/amp-a4a.js:864]: Do we need to cast it to boolean? (if undefined)

File: extensions/amp-a4a/0.1/amp-a4a.js [hunk 2]
```diff
@@ -179,15 +178,6 @@ const LIFECYCLE_STAGE_TO_ANALYTICS_TRIGGER = {
   'crossDomainIframeLoaded': AnalyticsTrigger.AD_IFRAME_LOADED,
 };
 
-/**
- * @const @enum {string}
- */
-export const NO_SIGNING_EXP = {
-  id: 'a4a-no-signing',
-  control: '21066324',
```
Comment (powerivq) [inline extensions/amp-a4a/0.1/amp-a4a.js:187]: If we remove this, where do we track the exp id?

**Review by powerivq — CHANGES_REQUESTED** [review]

File: extensions/amp-a4a/0.1/amp-a4a.js (hunk 2 above)
Comment (calebcordry) [reply extensions/amp-a4a/0.1/amp-a4a.js:187]: This is an RTV experiment so we just compare version numbers.

File: extensions/amp-a4a/0.1/amp-a4a.js (hunk 1 above)
Comment (calebcordry) [reply extensions/amp-a4a/0.1/amp-a4a.js]: Done.

**Review by powerivq — APPROVED** [review]

**Pull request merged**